README.md

```diff
@@ -728,6 +728,30 @@ In either case:
       end
     ```
 
+* <a name="unless-with-comparator"></a>Avoid `unless` with comparators if you can use `if` with an opposing comparator.<sup>[[link](#unless-with-comparator)]</sup>
```

Comment:
Well, we've had this discussion before and landed on the side of keeping it concise instead of wordy. If the rationale for this particular guideline isn't clear, maybe it shouldn't be included or we make an exception and include a very brief rationale here, as have in a few other cases.